### ISSUE TITLE
Getting "System.NullReferenceException" on calling IsDefined

### ISSUE THREAD

**Issue by Gniax** [xin9le/FastEnum#7]
Hello,
I'm just trying to check if an id exists before getting its value in my enum, however it throws me back a System.NullReferenceException.
I got a basic enum file (Apple = 1, Banana = 3, .....) and I'm just looking to get Key by Id.
Here's the line:
`bool res = FastEnum.IsDefined<ObjectsEnum>(key);`

Sample of my Enum class:
```
public enum ObjectsEnum
    {
        Epée_de_Boisaille = 44,
        Epée_de_Fouraille = 49,
        Epée_d733Alle = 55,
```
Thank you for your enum utility.

**Comment by xin9le** [comment]
I'm very sorry... The part you pointed out was degraded by the previous performance improvement commit (https://github.com/xin9le/FastEnum/commit/0d8a6c5e91eff5c0ae19f10ec51daf2f6d94b66e).

I have just released fixed version ([v1.5.3](https://github.com/xin9le/FastEnum/releases/tag/v1.5.3)). Please try it out.

**Comment by Gniax** [comment]
No problem ahahaha, yess it works, thank you very much!
